Update quickstart.md

mrezabidhendi opened the pull request:
The Variable `ABI` was defined in all CAPS, but it was used in the function call in all lowercase which lead to error.

## Description

Please include a summary of the changes and be sure to follow our [Contribution Guidelines](https://github.com/web3/web3.js/blob/4.x/.github/CONTRIBUTING.md).

<!--
Optional if an issue is fixed:
Fixes #(issue)
-->

## Type of change

<!-- Please delete options that are not relevant. -->

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran `npm run lint` with success and extended the tests and types if necessary.
- [ ] I ran `npm run test:unit` with success.
- [ ] I ran `npm run test:coverage` and my test cases cover all the lines and branches of the added code.
- [ ] I ran `npm run build` and tested `dist/web3.min.js` in a browser.
- [ ] I have tested my code on the live network.
- [ ] I have checked the Deploy Preview and it looks correct.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [ ] I have linked Issue(s) with this PR in "Linked Issues" menu.

mrezabidhendi committed:
Update quickstart.md

The Variable `ABI` was defined in all CAPS, but it was used in the function call in all lowercase which lead to error.

danforbes commented:
Thank you for catching this! I think it would actually be better to change the variable name to lowercase.

mrezabidhendi committed:
Merge branch '4.x' into patch-1

mrezabidhendi committed:
Update quickstart.md

Changed the variable "abi" to all lowercase both in the variable definition and in the "Contract" call

mrezabidhendi reviewed (commented):
Changed the `abi` variable to all lowercase both in variable definition and as the input in Contract method's call.

jdevcs commented:
@SantiagoDevRel changes mentioned in this PR should be merged via new PR created by you.
